projects/dotcms-field-elements/src/utils/errorHandling.tsx

```diff
@@ -0,0 +1,62 @@
+const fieldValidationMap = {
+    options: stringValidator,
+    regexCheck: regexValidator
+};
+
+interface PropValidationInfo<T> {
+    field: {
+        type: string;
+        name: string;
+    };
+    name: string;
+    value: T;
+}
+
+class DotFieldParamError<T> extends Error {
+    private readonly propInfo: PropValidationInfo<T>;
+
+    constructor(propInfo: PropValidationInfo<T>, expectedType: string) {
+        super(
+            `Warning: Invalid prop "${propInfo.name}" of type "${typeof propInfo.value}" supplied to "${propInfo
+                .field.type}" with the name "${propInfo.field.name}", expected "${expectedType}".
+        \nDoc Reference: https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/${propInfo
```

Comment:
You don't need `\n` when you are in a template string you can use break line as normal.

Reply:
it can be removed and added a break line, but if removed then it will look funny on the code..